docs/API_REFERENCE.md

```
@@ -34,41 +34,31 @@
-| Method | URI | Description | Request Body | Response Body | Status |
-|---|---|---|---|---|---|
-| POST | `/api/v1/groups` | 그룹 생성 | `GroupCreateRequestDto` <br> `{ name, description, imageUrl }` | `GroupCreateResponseDto` <br> `{ id, name, description, memberCount, profileImageUrl, inviteCode }` | ✅ Implemented |
-| POST | `/api/v1/groups/{groupId}/invite-codes` | 초대코드 생성 | `InviteCodeRequestDto` <br> `{ duration }` | `GroupInviteCodeResponseDto` <br> `{ inviteCode, expiresAt }` | ✅ Implemented |
-| POST | `/api/v1/groups/join` | 그룹 가입 (초대코드) | `GroupJoinRequestDto` <br> `{ inviteCode }` | `GroupIdResponseDto` <br> `{ id }` | ✅ Implemented |
-| POST | `/api/v1/groups/{groupId}/leave` | 그룹 탈퇴 | `(None)` | `ApiResponse` <br> `(void)` | ✅ Implemented |
-| GET | `/api/v1/groups/my` | 내 그룹 목록 조회 | `(None)` | `List<GroupResponseDto>` <br> `[{ id, name, description, memberCount, profileImageUrl }]` | ✅ Implemented |
-| GET | `/api/v1/groups/{groupId}` | 그룹 상세 조회 | `(None)` | `GroupResponseDto` <br> `{ id, name, description, memberCount, profileImageUrl }` | ✅ Implemented |
-
-## 4. Item API
+## 3. Item API
 | Method | URI | Description | Request Body | Response Body | Status |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)